pkg/local_object_storage/pilorama/interface.go

```diff
@@ -20,6 +20,8 @@ type Forest interface {
 	TreeGetByPath(cid *cidSDK.ID, treeID string, attr string, path []string, latest bool) ([]Node, error)
 	// TreeGetMeta returns meta information of the node with the specified ID.
 	TreeGetMeta(cid *cidSDK.ID, treeID string, nodeID Node) (Meta, error)
+	// TreeGetChildren returns children of the node with the specified ID. The order is arbitrary.
```

Comment:
According to implementations:

- Returns `ErrTreeNotFound` if tree is missing.
- Returns both `nil` if the given node has no children.

I suggest to clarify this in interface method's docs.

Reply:
Fixed.